simpleclient_servlet/src/main/java/io/prometheus/client/exporter/MetricsServlet.java

```diff
@@ -14,6 +14,23 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * The servlet exposes the metrics
+ *
+ * <p>Example config to export metrics on path {@code /metrics}:
```

Comment:
This makes no sense to me as it lacks context. What is it that takes this configuration?

As this is not tied to the servlet itself, I don't think this should be here.

Reply:
thanks for the feedback. tried to address that